packages/model/src/webhook.ts

```diff
@@ -7,8 +7,7 @@ export type EntityEventAction =
 
 export type EventEvent = `Events${EntityEventAction}`;
 export type ExternalAuthorEvent = `ExternalAuthors${EntityEventAction}`;
-export type GroupEvent = `Groups${EntityEventAction}`;
-export type InterestGroupEvent = `InterestGroups${EntityEventAction}`;
```

Comment:
This should be reverted as well I believe when we undo the changes in serverless